component/service/src/main/java/org/exoplatform/social/rest/impl/relationship/RelationshipsRestResourcesV1.java

```diff
@@ -236,9 +235,13 @@ public Response updateRelationshipById(@Context UriInfo uriInfo,
     if (type != null && ! type.equals(Relationship.Type.ALL)) {
       if (type.equals(Relationship.Type.IGNORED)) {
         relationshipManager.delete(relationship);
+        LogUtils.logInfo("relationships", "ignore-connection-request", "sender:" + relationship.getSender().getRemoteId() + ", receiver:" + relationship.getReceiver().getRemoteId(), LogUtils.STATUS_OK, this.getClass());
       } else {
         relationship.setStatus(type);
         relationshipManager.update(relationship);
+        if (type.equals(Relationship.Type.CONFIRMED)) {
+          LogUtils.logInfo("relationships", "confirm-connection-request", "sender:" + relationship.getSender().getRemoteId() + ", receiver:" + relationship.getReceiver().getRemoteId(), LogUtils.STATUS_OK, this.getClass());
```

Comment:
```suggestion
          LogUtils.logInfo("relationships", "confirm-connection-request", "sender:" + relationship.getSender().getRemoteId() + ",receiver:" + relationship.getReceiver().getRemoteId(), LogUtils.STATUS_OK, this.getClass());
```
Spaces are not avoided to avoid artifacts after the log parsing